library/Icingadb/ProvidedHook/Reporting/SlaReport.php

```diff
@@ -206,6 +207,11 @@ public function getHtml(Timerange $timerange, array $config = null)
         }
 
         $tableRows = [];
+        $configs = Config::fromIni(Config::resolvePath('modules/reporting/config.ini'));
```

Comment:
Huh? Report configuration is stored in database. That's controlled by `initConfigForm()`. Any element added there is available in `$config` afterwards.